docs/recipes/running-tasks-in-series.md

```diff
@@ -1,66 +1,95 @@
-# Running tasks in series, i.e. Task Dependency
+# Running tasks in series
 
-By default, tasks run with maximum concurrency -- e.g. it launches all the tasks at once and waits for nothing. If you want to create a series where tasks run in a particular order, you need to do two things:
+By default, gulp cli run task with maximum concurrency -- e.g. it launches all the tasks at once 
+and waits for nothing. If you want to create a series where tasks run in a particular order, 
+you should use `gulp.series`;
 
-- give it a hint to tell it when the task is done,
-- and give it a hint that a task depends on completion of another.
-
-For these examples, let's presume you have two tasks, "one" and "two" that you specifically want to run in this order:
+```js
+var gulp = require('gulp');
+var doAsyncStuff = require('./stuff');
 
-1. In task "one" you add a hint to tell it when the task is done. Either take in a callback and call it when you're done or return a promise or stream that the engine should wait to resolve or end respectively.
+gulp.task('one', function(done) {
+  doAsyncStuff(function(err){
+      done(err);
+  });
+});
 
-2. In task "two" you add a hint telling the engine that it depends on completion of the first task.
+gulp.task('two', function(done) {
+  // do things
+  done();
+});
 
-So this example would look like:
+//
+gulp.task('default', gulp.series('one', 'two'));
+```
 
+Another example, using a dependency pattern. It uses 
+[`async-once`](https://www.npmjs.com/package/async-once) to run the `clean`
+task operations only once:
 ```js
 var gulp = require('gulp');
-
-// takes in a callback so the engine knows when it'll be done
-gulp.task('one', function(cb) {
-    // do stuff -- async or otherwise
-    cb(err); // if err is not null and not undefined, the orchestration will stop, and 'two' will not run
+var del = require('del'); // rm -rf
+var once = require('async-once');
+
+gulp.task('clean', once(function(done) {
+  // Run only once.
+  // For the next call to the clean task, once will call done with 
+  // the same arguments than the first call.
```

Comment:
also gulp uses lowercase letters at the beginning of comments, not sure if that's something you care about